django/thunderstore/webhooks/models.py

```diff
@@ -60,7 +61,11 @@ def __str__(self):
     def get_for_package_release(cls, package):
         base_query = Q(Q(webhook_type=WebhookType.mod_release) & Q(is_active=True))
         community_query = Q()
+        community_exclusions = []
         for listing in package.community_listings.all():
+            if listing.review_status == PackageListingReviewStatus.rejected:
```

Comment:
I'm assuming there's no need to check whether `community.require_package_listing_approval` is True or not?

Reply:
Probably would be a good idea to check actually now that you mention it 👍 let me make changes